products/focus/testing_focus.md

```diff
@@ -10,13 +10,13 @@ Screenshots are provided by the mobile teams and sent out on dev-l10n. The lates
 
 ## Test builds
 
-For Focus Android, while there are Nightly builds available via the Google Play Store, the process is currently manual and localizers should reach out to the l10n mobile project manager in order to be added to these builds (currently **delphine** at **mozilla** dot **com**), and include in the request the email they use to connect to Google Play Store. Once the account has been whitelisted (usually the next day), localizers will need to opt-in with the same email they sent out, by going [to this URL](https://play.google.com/apps/internaltest/4698596548865727179). If the release version of the Focus app is already installed, it will update shortly after and be replaced by the Nightly version. If the app isn’t installed yet, the Nightly version will be dowloaded directly.
+There are Nightly localized builds for Focus for Android, that are accessible by following [the instructions here](https://github.com/mozilla-mobile/focus-android/wiki/Release-tracks). There are other builds available too, but Nightly is preferred for testing localizations, since it is updated daily.
 
-Note that for those who do not wish to use Google Play, there are also Nightly builds available on [Taskcluster here](https://tools.taskcluster.net/index/project.focus.android/preview-builds). The build to download is the APK file containing "universal-beta" in the name.
+Note that for those who do not wish to use Google Play, there are also Nightly builds available on [Taskcluster here](https://tools.taskcluster.net/index/project.mobile.focus.nightly/latest). The build to download is the APK file containing "universal-release" in the name.
 
-For Focus for iOS, please note an iOS device running v9 or above is needed. There is no specified version needed for Android.
+For Focus for iOS Nightly builds, reach out to the mobile project manager (currently delphine at mozilla dot com) to be added to the BuddyBuild system. There are also Beta builds [available here](https://www.mozilla.org/en-US/firefox/ios/testflight/).
 
-There are Beta builds available for Focus Android as well, that can be found by scrolling to the bottom of the [Focus for Android Play Store listing](https://play.google.com/store/apps/details?id=org.mozilla.focus&hl).
+For Focus for iOS, please note an iOS device running v9 or above is needed. There is no specified version needed for Android.
```

Comment:
I'm pretty sure there are for Android too. From the [Play Store page](https://play.google.com/store/apps/details?id=org.mozilla.focus): `Requires Android 5.0 and up`